packages/fxa-auth-server/lib/routes/oauth/index.js

```diff
@@ -278,6 +278,25 @@ module.exports = (log, config, oauthdb, db, mailer, devices) => {
         // done with 'session_token_id' at this point, do not return it.
         delete grant.session_token_id;
 
+        // attempt to record metrics, but swallow the error if one is thrown.
+        try {
+          let uid = sessionToken.uid;
```

Comment:
`let uid = sessionToken && sessionToken.uid` otherwise the `!uid` below will never hit